src/emulator/functionsEmulator.ts

```diff
@@ -147,14 +143,11 @@ export class FunctionsEmulator implements EmulatorInstance {
         }
       });
 
-      // TODO(samstern): Is this a race condition?  Could 'ready' happen before we're listening for it?
-      EmulatorLogger.log("DEBUG", `[functions] Waiting for runtime to be ready!`);
-      const log = await worker.waitForSystemLog((evt) => {
-        return evt.data.state === "ready";
-      });
-
       // For analytics, track the invoked service
-      track(EVENT_INVOKE, log.data.service);
+      if (triggerId) {
```

Comment:
By making this function not `static` we were able to just used the trigger definitions held on the emu instance rather than waiting for a log.